src/main/java/com/microsoftopentechnologies/windowsazurestorage/WAStorageClient.java

```diff
@@ -180,6 +180,22 @@ private static String getCustomURI(String storageAccountName, String type,
 	    return null;
 	}
     }
+    
+    /**
+     * Returns suffix for blob endpoint.
+     *
+     * @param blob endpoint
+     * @return the endpoint suffix
+     */
+    private static String getEndpointSuffix(String blobURL)
+    {
+    	int endSuffixStartIndex = blobURL.toLowerCase().indexOf(Utils.BLOB_ENDPOINT_ENDSUFFIX_KEYWORD);
+    	if(endSuffixStartIndex < 0) {
```

Comment:
no,  0 should be ok as if your input like core.windows.net , it should also work